### ISSUE TITLE
Incompatible for meteor version > 0.9.0

### ISSUE THREAD

**Issue by kaushik1979** [grandmore/meteor-ace-embed#2]
Are you doing any work on making the ace-embed compatible with the latest meteor released version - 0.9.2.2?

If you can tell me what all needs to be done for the compatibility, may be I can help you.

**Comment by vitch** [comment]
I have installed this alongside meteor 1.0 with the following command:

`meteor add mrt:ace-embed`

Then before creating an editor (with `ace.edit`) I do the following so that the runtime files are loaded from the correct place:

```
ace.config.set("modePath", "/packages/mrt_ace-embed/ace");
ace.config.set("themePath", "/packages/mrt_ace-embed/ace");
ace.config.set("workerPath", "/packages/mrt_ace-embed/ace");
ace.config.set("basePath", "/packages/mrt_ace-embed/ace");
```

(the paths have `mrt_` prepended - I guess because of a change in how the package is installed without meteorite)

I'm very new to meteor but the workaround meant I could get some legacy code working - hopefully it helps you :smile:

**Comment by dcsan** [comment]
right, all assets that are in packages now have the author's name prepended, and "mrt" is the author for this since it was auto-ported to 1.0 compatible.
this is a bit of a problem since if someone forked this editor, all the paths would have to be changed. as far as i know there isn't a way to get an author's name dynamically from the code?

**Comment by grandmore** [comment]
kaushik1979 - I am a bit busy at the moment but I was going to get around to it. I need to port my entire meteor app to version 1.0 which breaks all the packages I used. 

If you have time to send me what I need to do that would make it easier. Cheers.
Cheers

**Comment by JogoShugh** [comment]
This worked for me:

``` javascript
ace.config.set("modePath", "/packages/mrt_ace-embed/ace");
ace.config.set("themePath", "/packages/mrt_ace-embed/ace");
ace.config.set("workerPath", "/packages/mrt_ace-embed/ace");
ace.config.set("basePath", "/packages/mrt_ace-embed/ace");
```

I had lost all syntax highlighting and it was sad :)
